doc/source/rllib/rllib-learner.rst

```diff
@@ -0,0 +1,438 @@
+.. include:: /_includes/rllib/announcement.rst
+
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+.. |tensorflow| image:: images/tensorflow.png
+    :class: inline-figure
+    :width: 16
+
+.. |pytorch| image:: images/pytorch.png
+    :class: inline-figure
+    :width: 16
+
+
+Learner (Alpha)
+==================
+
+:py:class:`~ray.rllib.core.learner.learner.Learner` allows you to abstract the training 
+logic of RLModules. It supports both gradient-based and non-gradient-based updates (e.g.
+polyak averaging, etc.) The API is designed so that it can be distributed using data-
+distributed parallel (DDP). The Learner achieves the following:
```

Comment:
```suggestion
polyak averaging, etc.) The API is designed so that it can be distributed using data-distributed parallel (DDP). The Learner achieves the following:
```